crates/bevy_render/src/view/view.wgsl

```diff
@@ -8,13 +8,13 @@ struct ColorGrading {
 }
 
 struct View {
-    view_proj: mat4x4<f32>,
-    unjittered_view_proj: mat4x4<f32>,
-    inverse_view_proj: mat4x4<f32>,
-    view: mat4x4<f32>,
-    inverse_view: mat4x4<f32>,
-    projection: mat4x4<f32>,
-    inverse_projection: mat4x4<f32>,
+    world_to_clip: mat4x4<f32>,
+    unjittered_world_to_clip: mat4x4<f32>,
+    clip_to_world: mat4x4<f32>,
+    view_to_world: mat4x4<f32>,
+    world_to_view: mat4x4<f32>,
+    view_to_clip: mat4x4<f32>,
+    clip_to_view: mat4x4<f32>,
```

Comment:
@DGriffin91 and I have been discussing doing this in a thread in #rendering-dev on Discord. Clip space is a 4D homogeneous space before perspective division. After the perspective division we have the 3D normalised device coordinate space. So the problem we have been discussing is that clip_to_world and clip_to_view are technically ndc_to_world_homogeneous and ndc_to_view_homogeneous, and then to get world or view, you need to take the xyz components and do the perspective division by the w component. We were discussing adding helper functions to do this that would be named position_ndc_to_world() and position_ndc_to_view(), for example.

I am just passing through so don't have time to decide on this right now. As such I'm going to mark this review as request for changes to make clear that a decision needs to be made about this particular issue.